hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/HapiTxnOp.java

```diff
@@ -577,6 +577,16 @@ public T memo(String text) {
         return self();
     }
 
+    public T blankMetadata() {
+        memo = Optional.of("");
```

Comment:
Should this be `metadata` instead of `memo`?

Reply:
I think @mhess-swl is right. Please double-check and eventually fix it in a follow-up PR.